package.json

```diff
@@ -29,6 +29,8 @@
     "lint:scripts": "eslint app/scripts/",
     "lint:css": "stylelint 'app/styles/**/**' 'app/scripts/**/*.(js|ts|tsx|jsx)'",
     "postinstall": "husky",
```

Comment:
Thank you for handling this swiftly. Can you explain me why we need hustky for postinstall command?

Reply:
It is a quirk to how husky operates. Running that configure command essentially installs a husky script that doesn't actually get committed on the developer’s machine. Another option would just be to tell people as part of the setup to install husky. It's a little more graceful for using npm, but for yarn you have to do this post install.

They talk about it briefly in this issue:
https://github.com/typicode/husky/issues/1016

A third option would be to enforce this with a different package than husky such as https://www.npmjs.com/package/lefthook